### DISCUSSION TITLE
not able to execute git commit

### DISCUSSION THREAD

**Discussion by shreelakshmijoshi** [atom/atom#22474] [Q&A]
I made atom as default editor in git bash by executing this
`git config --global core.editor "atom --wait"`
checked the git configuration and saw `editor = atom --wait`
But when I execute git commit in git bash, I get this

> hint: Waiting for your editor to close the file... /c/Users/shree/AppData/Local/atom/bin/../app-1.57.0/resources/cli/atom.sh: line 23: cmd.exe: command not found

Aborting commit due to empty commit message.

I don't know what to do.. Can somebody please help ?

**Comment by icecream17** [comment]
In the file in the stacktrace, atom does `cmd.exe ATOMCMD something`

Maybe it would be possible to debug what `ATOMCMD` is, by editing `/c/Users/shree/AppData/Local/atom/bin/../app-1.57.0/resources/cli/atom.sh` - but I don't know much about bash

https://github.com/atom/atom/blob/ea4cbb7e8433efdd74d969fe89b00d8470e8c58a/resources/win/atom.sh#L22:L26

I'm not sure where "Aborting commit due to empty commit message" comes from.

**Reply by shreelakshmijoshi** [reply]
Oh okay.. I will try implementing the given solution, though changing the internal shell commands seems scary as I am a newbie.
Thanks a lot for the help : )

**Reply by shreelakshmijoshi** [reply]
update , I tried adding the message with the commit and it worked !

> $ git commit -m 'Adding basic html, css, js files'
[master (root-commit) 8ad6d32] Adding basic html, css, js files
 3 files changed, 14 insertions(+)
 create mode 100644 css/app.css
 create mode 100644 index.html
 create mode 100644 js/app.js

**Comment by shreelakshmijoshi** [comment]
I tried adding commit with the message , and it seems to work fine.
`$ git commit -m 'Adding basic html, css, js files'`

> [master (root-commit) 8ad6d32] Adding basic html, css, js files
>  3 files changed, 14 insertions(+)
>  create mode 100644 css/app.css
>  create mode 100644 index.html
>  create mode 100644 js/app.js

But my code editor does not open when I commit, to type out my commit message.
